Makefile

```
@@ -44,10 +44,10 @@ markdownlint:
 .PHONY: install-yamllint
 install-yamllint:
     # Using a venv is recommended
-	pip install -U yamllint~=1.30.0
+	yamllint --version >/dev/null 2>&1 || pip install -U yamllint~=${YAMLLINT_VERSION}
```

```suggestion
	yamllint --version >/dev/null 2>&1 || pip install -U yamllint~=$(YAMLLINT_VERSION)
```

Should we use Makefile syntax instead of the Bash one? I would use the Makefile one to keep it align with the other Makefile's variable substitution